### PR TITLE
KEYCLOAK-9387: Add hor scroll & tooltips to role selectors

### PR DESCRIPTION
**Pull request by ssilvert**

### CODE REVIEW

**Review by mposolda — APPROVED** [review]
Thanks for the fix!

I left just one comment/question, but it seems it will probably not require any changes in the PR.

For now, I am adding the label "Hold" as AFAIK the master is still not yet opened for fixes (besides test related fixes).

File: themes/src/main/resources/theme/base/admin/resources/js/services.js [hunk 1]
```diff
@@ -196,6 +196,13 @@ module.factory('ComponentUtils', function() {
 
     var utils = {};
 
+    utils.findIndexById = function(array, id) {
```
[the remark this answers is not in the document]
Comment (ssilvert) [reply themes/src/main/resources/theme/base/admin/resources/js/services.js]: Or to be more specific, the object's elements are the same but it is not the same object instance, so indexOf doesn't work.

[the remark this answers is not in the document]
Comment (mposolda) [reply themes/src/main/resources/theme/base/admin/resources/js/services.js]: @ssilvert ah, ok. Thanks!

**Comment by pdrozd** [comment]
@keycloak-ci-bot test

**Review by pdrozd — APPROVED** [review]

**Pull request merged**